HaxmLoader/haxm_loader.c

```diff
@@ -110,9 +156,8 @@ static int DriverInstall(const wchar_t *sysFilePath)
 
     pathLen = GetFullPathNameW(sysFilePath, MAX_PATH, driverLocation, NULL);
     if (!pathLen || pathLen >= MAX_PATH) {
-        printf("%s(): Failed to get absolute path:\r\n",
-                __FUNCTION__);
-        wprintf(L"  sysFilePath='%s'\r\n", sysFilePath);
+        wprintf(L"%s(): Failed to get absolute path:\r\n  sysFilePath=\"%s\"\r\n",
```

Comment:
Can wchar_t string be split into multiple lines as this line exceeds 80 columns?

Reply:
Sure, will do that.